Dockerfile

```
@@ -41,10 +41,10 @@ RUN apk update && apk add wget && wget -O  "apache-tomcat-${TOMCAT_VERSION}.tar.
 
 
 # registry.cloudogu.com/official/cas
-FROM registry.cloudogu.com/official/java:11.0.19-1
+FROM registry.cloudogu.com/official/java:11.0.20-1
 
 LABEL NAME="official/cas" \
-      VERSION="6.6.10-1" \
+      VERSION="6.6.12-1" \
```

```suggestion
      VERSION="6.6.10-1" \
```
The version change will be handled by the release script.